src/Miew.js

```diff
@@ -3749,6 +3790,37 @@ Miew.prototype.scale = function (factor) {
   this._needRender = true;
 };
 
+/**
+ * Center view on selection
+ * @param {subset | string} selector - translation value (Ang) along model's X axis
+ */
+Miew.prototype.center = function (selector) {
+  if (selector === undefined) { // on current sellection
+    this.setPivotSubset();
+    this._needRender = true;
+    return;
+  }
+  if (selector.obj !== undefined && ('atom' in selector.obj || 'residue' in selector.obj)) { // from event with selection
```

Comment:
Here I propose to divide one conditional part into two: one for atom and one for residue, then replace the whole part after string selector processing. It will remove the need of check for (selector.obj !== undefined)